app/code/Magento/Review/Block/Adminhtml/Edit/Form.php

```diff
@@ -75,18 +75,23 @@ protected function _prepareForm()
         $review = $this->_coreRegistry->registry('review_data');
         $product = $this->_productFactory->create()->load($review->getEntityPkValue());
 
+        $formActionParam['id'] = $this->getRequest()->getParam('id');
+        $formActionParam['ret'] = $this->_coreRegistry->registry('ret');
+        if ($this->getRequest()->getParam('productId') != null) {
```

Comment:
```suggestion
        if ($this->getRequest()->getParam('productId')) {
```

Same for `customerId`

Reply:
Hello @orlangur 
I have updated PR as your suggestion can you review it.